src/Routes.js

```diff
@@ -104,50 +50,121 @@ export const Routes = () => {
     }
   }, [hasSystems]);
   return (
-    <Suspense
-      fallback={
-        <Bullseye>
-          <Spinner size="xl" />
-        </Bullseye>
-      }
-    >
-      {!hasSystems ? (
-        <Suspense
-          fallback={
-            <Bullseye>
-              <Spinner size="xl" />
-            </Bullseye>
-          }
-        >
-          <AsynComponent
-            appId={'advisor_zero_state'}
-            appName="dashboard"
-            module="./AppZeroState"
-            scope="dashboard"
-            ErrorComponent={<ErrorState />}
-            app="Advisor"
-          />
-        </Suspense>
-      ) : (
-        <Switch>
-          {paths.map((path) => (
-            <Route
-              key={path.title}
-              path={path.path}
-              exact
-              component={path.component}
-              rootClass={path.rootClass}
-            />
-          ))}
-          <AdminProtectedRoute
-            exact
-            path="/topics/admin/manage"
-            component={() => <TopicAdmin />}
-          />
-          {/* Finally, catch all unmatched routes */}
-          <Redirect path="*" to="/recommendations" />
-        </Switch>
-      )}
-    </Suspense>
+    <Routes>
+      <Route
+        key={'Recommendations'}
+        path="/recommendations"
+        element={
+          <ZeroState check={hasSystems}>
```

Comment:
Why not wrap the `Routes` containing all `Route` components in one `ZeroState`?

Reply:
Agree, was waiting for other pair of eyes to look at this zerostate I wrote to see if it any good